k8s/environments/prod/cluster-00/mi/mi-extraction-service-recover-data.yaml

```diff
@@ -0,0 +1,76 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: mi-extraction-service
+  namespace: mi
+  annotations:
+    flux.weave.works/automated: "true"
+    repository.fluxcd.io/job: job.image
+    filter.fluxcd.io/job: glob:prod-*
+    repository.fluxcd.io/smoke: job.smoketests.image
+    filter.fluxcd.io/smoke: glob:prod-*
+spec:
+  releaseName: mi-extraction-service
+  chart:
+    git: git@github.com:hmcts/shared-services-flux
+    path: k8s/release/mi/mi-extraction-service
+    ref: master
+  values:
+    job:
+      image: sdshmctspublic.azurecr.io/mi-extraction-service:prod-2c2a120
+      suspend: false
+      kind: Job
+      activeDeadlineSeconds: 82800
+      memoryRequests: '2048Mi'
+      cpuRequests: '200m'
+      memoryLimits: '4096Mi'
+      cpuLimits: '2000m'
+      labels:
+        app.kubernetes.io/instance: mi-extraction-service
+        app.kubernetes.io/name: mi-extraction-service
+      keyVaults:
+        "mi-vault":
+          excludeEnvironmentSuffix: false
+          resourceGroup: mi-{{ .Values.global.environment }}-rg
+          secrets:
+          - appinsights-instrumentationkey
+          - sftp-remote-user
+          - sftp-remote-password
+          - encryption-publicKey
+      environment:
+        APPINSIGHTS_INSTRUMENTATIONKEY: "8bfa6e68-ae68-442c-a237-83cde3b092f3"
+        MI_CLIENT_ID: e8a52cff-ef0c-4447-baf2-665c3480d0de
+        SFTP_ENABLED: true
+        SFTP_FORCECHECK: true
+        SFTP_FILE_ENCRYPTION_ENABLED: true
+        SFTP_REMOTE_HOST: 10.24.254.130
+        SFTP_REMOTE_PORT: 22
+        SFTP_REMOTE_FOLDER: /files/mi-platform-export/
+        RETRIEVE_FROM_DATE: 2021-03-35
```

Comment:
2021-03-24

Reply:
Using 2021-03-25 as the From date as well otherwise it'll extract 24th data as well.